vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -70,6 +75,14 @@
           - clusterctl.cluster.x-k8s.io=""
           - cluster.x-k8s.io/provider="metal3"
     when: CAPM3_VERSION == "v1alpha3"
+  
+  - name: Ironic TLS setup 
+    shell: "source ./{{ metal_dir }}/lib/ironic_tls_setup.sh"
+    when: IRONIC_TLS_SETUP == "true"
+
+  - name: Ironic basic authentification setup
+    shell: "source ./{{ metal_dir }}/lib/ironic_basic_auth.sh"
+    when: IRONIC_BASIC_AUTH == "true"
```

Comment:
this does nto do anything, since you do not do anything in the shell after sourcing the file

Reply:
Moved them up before installing ironic